docs/sources/configuration/_index.md

```diff
@@ -2208,6 +2204,10 @@ The `limits_config` block configures global and per-tenant limits in Loki.
 # CLI flag: -store.max-query-length
 [max_query_length: <duration> | default = 721h]
 
+# Address, including port, where the compactor api is served
+# CLI flag: -store.compactor-address
+[compactor_address: <string> | default = ""]
+
```

Comment:
It seems the documentation has been added under `limits_config` instead of `storage_config`.